packages/x-date-pickers/src/internals/utils/text-field-helper.test.ts

```diff
@@ -46,12 +70,7 @@ describe('text-field-helper', () => {
       if (isValid) {
         expect(runMaskValidation()).to.be.equal(true);
       } else {
-        expect(runMaskValidation).toWarnDev(
-          [
-            `The mask "${mask}" you passed is not valid for the format used ${format}.`,
-            `Falling down to uncontrolled no-mask input.`,
-          ].join('\n'),
-        );
```

Comment:
I removed the `The mask ... you passed` because when masked is inferred, it does not make sens to show this error message to the dev